src/util/options.schema.json

```diff
@@ -2099,6 +2099,12 @@
           "type": "boolean",
           "default": true
         },
+        "nonTerminating": {
+          "title": "warn.nonTerminating",
+          "description": "Non Termination warning",
+          "type": "boolean",
+          "default": true
+        },
```

Comment:
This naming is inconsistent with the rest. `warn.termination` seems good enough. We don't have separate categories for negations, that's what we use message severities for.